tests/async/test_async_auth.py

```diff
@@ -60,15 +61,52 @@ async def test_alogin_new_user(self):
         self.assertIsInstance(user, User)
         self.assertEqual(user.username, second_user.username)
 
-    async def test_alogin_without_user(self):
+    async def test_alogin_without_user_no_request_user(self):
```

Comment:
```suggestion
    # RemovedInDjango61Warning: When the deprecation ends, replace with:
    # async def test_alogin_without_user(self):
    async def test_alogin_without_user_no_request_user(self):
```
I think when we get rid of the code we only need one test for the case `user=None`